pages/index.js

```diff
@@ -1,77 +1,47 @@
 import PlantList from "@/components/PlantList";
 import React, { useEffect, useState } from "react";
-import FilterForm from "@/components/FilterForm";
 import SearchField from "@/components/SearchField";
+import { StyledHeadline } from "@/components/Headline/StyledHeadline";
 import styled from "styled-components";
 
 export default function HomePage({ onToggleFavorite, favorites, plants }) {
   const [search, setSearch] = useState("");
-  const [filteredPlants, setFilteredPlants] = useState(plants);
-  const [counterMessage, setCounterMessage] = useState('');
-
-  function handleFilterUpdate(newFilteredPlants) {
-    setFilteredPlants(newFilteredPlants);
-  }
-
-const searchResult = plants.filter((plant) => {
-  return plant.commonName.toLowerCase().startsWith(search.toLowerCase());
-});
-
-useEffect(() => {
-  if (searchResult.length > 0 && searchResult.length < plants.length) {
-    setCounterMessage(
-      `Showing ${searchResult.length} of ${plants.length} plants:`
-    );
-  } else if (filteredPlants.length === plants.length && searchResult.length === plants.length) {
-    setCounterMessage('');
-  }
-}, [searchResult]);
-
-useEffect(() => {
-  if (filteredPlants.length > 0 && filteredPlants.length < plants.length) {
-    setCounterMessage(
-      `Showing ${filteredPlants.length} of ${plants.length} plants:`
-    );
-  } else if (filteredPlants.length === plants.length && searchResult.length === plants.length) {
-    setCounterMessage('');
-  }
-}, [filteredPlants]);
+  const [counterMessage, setCounterMessage] = useState("");
+
+  const searchResult = plants.filter((plant) => {
+    return plant.commonName.toLowerCase().startsWith(search.toLowerCase());
+  });
+
+  useEffect(() => {
+    if (searchResult.length > 0 && searchResult.length < plants.length) {
+      setCounterMessage(
+        `Showing ${searchResult.length} of ${plants.length} plants:`
+      );
+    } else if (searchResult.length === plants.length) {
+      setCounterMessage("");
+    }
+  }, [searchResult, plants.length]);
```

Comment:
As this is not part of this feature, I just want to add that I think that a `useEffect` might not be necessary for handling `counterMeesage` state. But you can think about this later during a refactoring. No need to change it now. 👍